example/image-classification/train_inception-resnet-v2-17flowers.py

```diff
@@ -0,0 +1,95 @@
+import find_mxnet
+import mxnet as mx
+import argparse
+import os, sys
+import train_model
+
+parser = argparse.ArgumentParser(description='train an image classifer on cifar10')
+parser.add_argument('--data-dir', type=str, default='./17flowers/',
+                    help='the input data directory')
+parser.add_argument('--gpus', type=str,
+                    help='the gpus will be used, e.g "0,1,2,3"')
+parser.add_argument('--num-examples', type=int, default=952,
+                    help='the number of training examples')
+parser.add_argument('--batch-size', type=int, default=16,
+                    help='the batch size')
+parser.add_argument('--data-shape', type=str, default='3,299,299',
+                    help='the data shape, e.g "3,299,299"')
+parser.add_argument('--lr', type=float, default=.001,
+                    help='the initial learning rate')
+parser.add_argument('--lr-factor', type=float, default=1,
+                    help='times the lr with a factor for every lr-factor-epoch epoch')
+parser.add_argument('--lr-factor-epoch', type=float, default=1,
+                    help='the number of epoch to factor the lr, could be .5')
+parser.add_argument('--model-prefix', type=str,
+                    help='the prefix of the model to load')
+parser.add_argument('--save-model-prefix', type=str,
+                    help='the prefix of the model to save')
+parser.add_argument('--num-epochs', type=int, default=700,
+                    help='the number of training epochs')
+parser.add_argument('--load-epoch', type=int,
+                    help="load the model on an epoch using the model-prefix")
+parser.add_argument('--kv-store', type=str, default='local',
+                    help='the kvstore type')
+args = parser.parse_args()
+
+# download data if necessary
+def _download(data_dir):
```

Comment:
I can delete the train_inception-resnet-v2-17flowers.py, but I must modify the train_imagenet.py because my model need the input_data_shape to be a param in the get_symbol, is it ok ? Pls tell me how to do with it then i will give an new commit.

Thanks again for this wonderful Framework